amplitude.js

```diff
@@ -336,6 +373,11 @@ Amplitude.prototype.logEvent = function(eventType, eventProperties) {
     localStorage.setItem(LocalStorageKeys.LAST_EVENT_TIME, this._lastEventTime);
     localStorage.setItem(LocalStorageKeys.LAST_EVENT_ID, eventId);
 
+    // Add the utm properties, if any, onto the user properties.
+    var userProperties = {};
+    object.merge(userProperties, this.options.userProperties || {}, this._utmProperties);
+    object.merge(userProperties, this._utmProperties);
```

Comment:
Yes, the but the 3rd param on the previous line should be removed. object.merge only merges 2 params at a time.